### PR TITLE
roughly finish the whole pages

### PR DESCRIPTION
**Pull request by lucy41033**
Dear I had roughly finished the whole pages, and will detailly fix every page in this couple days.

### CODE REVIEW

File: src/app/payment/payment.component.html [hunk 1]
```diff
@@ -3,32 +3,468 @@
...
+                        <label for="Award miles (Miles & More)" class="ml-3">Award miles (Miles & More)</label>
+                    </div>
+                </div>
+            </p-card>
+        </div>
+        <div style="padding-bottom: 20px ;">
+            <p-card header="Confirmation">
+                <hr style="border: 0.1px solid rgb(240, 236, 236)">
+                <span style="color: red;">Please confirm</span><br><br>
+                <p-checkbox formControlName="checkConfirm" value="b" inputId="b"></p-checkbox>
```
Comment (CYHsu9601) [inline src/app/payment/payment.component.html]: 如果目前沒有用到formControlName，建議先刪掉 formControlName="checkConfirm"，不然程式會報錯

File: src/app/payment/payment.component.html [hunk 2]
```diff
@@ -3,32 +3,468 @@
...
+                <label for="b">I was able to find out about the general terms and conditions of the tour operator
+                    holidays.ch GmbH and the agency </label>
+                <br> <br>
+                <p-checkbox formControlName="checkConfirm" value="a" inputId="a"></p-checkbox>
```
Comment (CYHsu9601) [inline src/app/payment/payment.component.html]: 同上建議

**Review by CYHsu9601 — COMMENTED** [review]
Please check below comments.

File: src/app/payment/payment.component.html (hunk 1 above)
Comment (CYHsu9601) [reply src/app/payment/payment.component.html]: image

File: src/app/payment/payment.component.html (hunk 2 above)
Comment (lucy41033) [reply src/app/payment/payment.component.html]: 刪掉formControlName="checkConfirm"囉

File: src/app/payment/payment.component.html (hunk 1 above)
Comment (lucy41033) [reply src/app/payment/payment.component.html]: 刪掉formControlName="checkConfirm"囉

**Review by CYHsu9601 — APPROVED** [review]
Run successfully

**Pull request merged**